src/main/resources/default/assets/common/scripts/slider.js.pasta

```diff
@@ -9,6 +9,9 @@
      * @param target  css selector selecting the outer html node containing all slides as child nodes
      * @param options optional object specifying the behaviour of the slider (see constants for possible values):
      *                 - animation:  Time in ms between sliding (can be undefined for no animation)
```

Comment:
if `options` is an object, and therefore `animation` is a member it could be documented as:
`@param {number} options.animation Time in ms between sliding (....)` also if `options.animation` is optional it could be wrapped in brackets `[options.animation]`. If there is a default value it could be specified with an equals such as: `[options.animation=100]`